packages/node_modules/@node-red/editor-api/lib/admin/plugins.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
         let opts = {
             user: req.user,
             module: req.params[0],
```

Comment:
Note that if the path is scoped without the plugin name, the first parameter is the scope. This is not correct.
e.g. `/plugins/@me/my-plugin-module/my-plugin`